common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -599,14 +599,7 @@ def close_connections(self):
         """
         Closes any open connections to the underlying database
         """
-        self.collection.database.connection.close()
-
-    def mongo_wire_version(self):
-        """
-        Returns the wire version for mongo. Only used to unit tests which instrument the connection.
-        """
-        self.database.connection._ensure_connected()
-        return self.database.connection.max_wire_version
+        self.collection.database.client.close()
```

Comment:
@giovannicimolin, is it correct to drop the return statement? I see this issue on the upstream PR as well.

Reply:
@lgp171188 I didn't remove the return statement of this function, I completely removed ` def mongo_wire_version(self):`, check the complete diff to see.
The `max_wire_version` isn't relevant when using `MongoClient`.

Reply:
@giovannicimolin, oops, sorry! I missed that the function itself had been removed and that the close statement was added to the previous function.